src/account/UpgradeableModularAccount.sol

```diff
@@ -360,18 +395,27 @@ contract UpgradeableModularAccount is
             revert AlwaysDenyRule();
         }
 
-        FunctionReference userOpValidationFunction = getAccountStorage().selectorData[selector].validation;
+        FunctionReference userOpValidationFunction = FunctionReference.wrap(bytes21(userOp.signature[:21]));
```

Comment:
I think I'll modify it elsewhere to not specify a starting index of 0 in a calldata slice, because in a few places we use `:]` to indicate the rest of the slice, and I'd prefer not to explicitly fetch and set the length in those cases.

Reply:
Sweet! Mind adding this to our style guide?